src/rwa003/config.sol

```diff
@@ -0,0 +1,55 @@
+// The following addresses are from a json file called: ns2-mainnet.json
+
+pragma solidity >=0.5.12;
+
+// Autogenerated Solidity Tinlake Address Contract 
+contract POOL_CONFIG {
+
+    // precision
+    uint256 constant public THOUSAND = 10 ** 3;
+    uint256 constant public MILLION  = 10 ** 6;
+    uint256 constant public WAD      = 10 ** 18;
+    uint256 constant public RAY      = 10 ** 27;
+    uint256 constant public RAD      = 10 ** 45;
+
+    // Maker changelog 
+    address constant MAKER_CHANGELOG = 0xdA0Ab1e0017DEbCd72Be8599041a2aa3bA7e740F;
+
+    // Tinlake addresses CF4 
+    address constant ROOT = 0x792164b3e10a3CE1efafF7728961aD506c433c18;
+    address constant DROP = 0x931C3Ff1F5aC377137d3AaFD80F601BD76cE106e;
+    address constant MGR = 0x45e17E350279a2f28243983053B634897BA03b64;
+    address constant RESERVE = 0x8d9466346ea306F43b9203F53140C3EB6777a75F;
+    address constant MEMBERLIST = 0xb7ee04cb62bFD87862e56E2E880b9EeB87aDf20F;
+    address constant COORDINATOR = 0xb9575aD050263cC0A9E65B8bd6041DbF5e02bf1F; 
+    address constant SENIOR_OPERATOR = 0xDeb6eEEF90bbb5be6A771250eb9bA8d0804c3F5D;
+    address constant ASSESSOR = 0xe5B1C9DB5316350e1501ADe417cBe44d2D6A04Cd;
+    address constant TRANCHE = 0x3bCe1712d1AaC8C9597Bc65F1c1630aF32F918B0;
+
+
+    // MIP21 contracts RWA003-A
+    address public constant MCD_JOIN = 0x4CCc7fED3912A32B6Cf7Db2FdA1554a9FF574099;
+    address constant GEM = 0xDBC559F5058E593981C48f4f09fA34323df42d51;
```

Comment:
This contract needs to be verified.

Reply:
Just verified this one (and therefore all GEMs). Will double check the rest as well.